CHANGELOG.md

```diff
@@ -30,8 +30,13 @@ You can check your current version with the following command:
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
 
-## 0.37.0a5
+## 0.37.0a6
+
+**March 13, 2024**
 
+- Adjusted `host.py`, `tools.py`, `test_tools.py`, `storage.py`, `test_storage.py`, and `fixtures_storage.py` in accordance with Pylint checks.
+
+## 0.37.0a5
```

Comment:
Master merge is needed

Reply:
I think it already has the latest changes.
Am I missing something?

Reply:
Sorry, my bad, confused due to diff visualization